src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow weak
```

Comment:
For instance, I think you could add `@flow weak` everywhere in this codebase without raising much issues. The upside would be that flow could still emit warnings on very risky things it managed to infer.